apps/.storybook/config.js

```diff
@@ -103,6 +104,19 @@ storybook.setAddon({
   }
 });
 
+storybook.setAddon({
+  withReduxStore(reducers = {}) {
```

Comment:
It'd be nice to have a comment describing this addon and maybe documenting its usage pattern.